### PR TITLE
Update slide down nav icons

### PR DESCRIPTION
**Pull request by martypowell**
- Updated the icons
- Updated the images to match production and for use with polaris

### CODE REVIEW

File: inclusions/slide-down-nav/account-get-a-library-card-section.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <div class="hidden-sm hidden-xs">
 	<h3>Get a Library Card</h3>
 	<p class="emphasize-text">A library card allows you to:</p>
-	<img class="pull-image-right" src="images/drawers/placeholder.jpg" />
+	<img class="pull-image-right hidden-sm hidden-xs hidden-md" src="//baltimorecountymd.gov/sebin/b/r/account-get-a-library-card.jpg" />
```
[the remark this answers is not in the document]
Comment (mxsnyder) [reply inclusions/slide-down-nav/account-get-a-library-card-section.html]: Ahh. Wise decision. Any reason for not using `//staging.bcpl.info/sebin/b/r/account-get-a-library-card.jpg`?

[the remark this answers is not in the document]
Comment (martypowell) [reply inclusions/slide-down-nav/account-get-a-library-card-section.html]: I Iz not thinking, let me change that.

**Comment by martypowell** [comment]
@mxsnyder ready for another look

**Comment by mxsnyder** [comment]
yasss

**Pull request merged**